src/messenger-shortcuts.js

```diff
@@ -127,7 +155,21 @@ function openHelp() {
   mute();
   setTimeout(function() {
     document.querySelector('div[role="dialog"] h2').innerHTML = "Keyboard Shortcuts for Messenger";
-    document.querySelector('div[role="dialog"] h2~div').innerHTML = HELP_HTML;
-    document.querySelector('div[role="dialog"] h2~div~div').remove();
-  }, 100);
+      document.querySelector('div[role="dialog"] h2~div').innerHTML = HELP_HTML;
+      document.querySelector('div[role="dialog"] h2~div~div').remove();
+    }, delay);    
+  delay = 0; // to prevent unnecessary timeout
+}
+function sendLike() {
```

Comment:
Holding down Alt+Shift+E sends a never-ending sequence of likes, which is probably not desirable. Any way to fix this?